src/modules/account/RadixTokenDefinitionAccountSystem.ts

```diff
@@ -98,12 +98,10 @@ export class RadixTokenDefinitionAccountSystem implements RadixAccountSystem {
         tokenDefinition.granularity = particle.granularity
 
         const mintPermissions = particle.permissions.mint
-        if (mintPermissions === RadixTokenPermissionsValues.SAME_ATOM_ONLY || mintPermissions === RadixTokenPermissionsValues.GENESIS_ONLY) {
+        if (mintPermissions === RadixTokenPermissionsValues.TOKEN_CREATION_ONLY || mintPermissions === RadixTokenPermissionsValues.NONE) {
             tokenDefinition.tokenSupplyType = RadixTokenSupplyType.FIXED
-        } else if (mintPermissions === RadixTokenPermissionsValues.TOKEN_OWNER_ONLY) {
+        } else if (mintPermissions === RadixTokenPermissionsValues.TOKEN_OWNER_ONLY || mintPermissions === RadixTokenPermissionsValues.ALL) {
```

Comment:
What does `RadixTokenPermissionsValues.ALL` mean? Is it like a super user (who has all permissions)?

Reply:
It means anyone can mint it, like the POW token currently(though I think we're making it into something else)

Reply:
Ok, but is that what we want to do? Allow anyone, not only the creator, to mint new tokens? I suppose it's an interim solution.

Reply:
If you create a token with that permission, yeah. It's not really exposed through the library, and I'm not sure if there are many use cases for that, but I don't see why not allow such an option.

Reply:
Correction, it is possible to make such a token through the js lib, but it's not trivial, you have to know what you're doing to do it